numpy/core/src/private/ufunc_override.c

```diff
@@ -306,12 +340,26 @@ PyUFunc_CheckOverride(PyUFuncObject *ufunc, char *method,
         goto fail;
     }
 
+    Py_ssize_t len = PyTuple_GET_SIZE(normal_args);
+    PyObject *override_args = PyTuple_New(len + 2);
```

Comment:
Really need to check for NULL here. Then you will want Py_XDECREF in failure path.